### ISSUE TITLE
Compile errors

### ISSUE THREAD

**Issue by cbreezier** [stormcloud-dev/stormcloud#1]
Is there any chance that this project will be revived or continued? Currently there are a lot of compile errors such as unfinished statements and mismatching arguments.

**Comment by renbinden** [comment]
I'll have a look at doing so in the near future - I have a lot of projects I work on in the limited free time I have, but considering how far this got, it would definitely be worth finishing.

[Update] I had a look at my local version today and I'm midway through implementing a lot of objects. Considering the game has about 520 objects, implementing them all will take a long time. The player's step method is progressing a lot, but there's a lot of complex interactions with other objects.

**Comment by sky0hunter** [comment]
Yep, I would like to finish it too, especially because my game server list
is down now, and theres no good hosting alternative at the moment.

**Comment by n1zzo** [comment]
Hi,
does any of the developers have a stable branch to push?
It does not matter if it's not feature complete but it would be nice to have something that compiles successfully at least.
